api/src/main/java/net/kyori/adventure/text/SelectorComponent.java

```diff
@@ -26,7 +26,12 @@
 import org.checkerframework.checker.nullness.qual.NonNull;
 
 /**
- * A selector component.
+ * A component that can display the name of entities found with a given selector.
```

Comment:
Who is the origin of the selector (i.e., if I `dualspiral` used `@s` and sent a message to you `KingOfSquares`, whose name would it display)?

Reply:
after talking with zml I'm going to clarify which components gets server-side rendering, since what you are asking is determined by the platforms(implementation detail).